Cairo Desktop/Cairo Desktop/CairoSettingsWindow.xaml

```diff
@@ -172,6 +177,10 @@
                         Name="chkEnableAutoUpdates">
                         <Label Content="{Binding Path=(l10n:DisplayString.sSettings_General_UpdateCheck)}" />
                     </CheckBox>
+                    <CheckBox Click="ChkRunAtLogOn_Click"
+                        x:Name="chkRunAtLogOn" IsChecked="{Binding RunAtLogOn}">
+                        <Label Content="{Binding Path=(l10n:DisplayString.sRunAtLogon)}" />
```

Comment:
The setting should only be visible if Cairo is not currently the running shell. You can view the other code that checks this for reference.